src/clap-saw-demo-editor.cpp

```diff
@@ -117,7 +117,14 @@ void ClapSawDemo::guiDestroy() noexcept
 #endif
 
     if (editor)
+    {
+        auto frame = editor->getFrame();
```

Comment:
Ah this is a good change but if you look right above it I swear about linux and frame closing and VSTGUI. This will crash linux right now in an annoying way. So

1. Can you move it to before the removeLinxuVSTGUIPlugin call above (where I have the close) and
2. Condition it on not linux

so it should be halt' if not linux close; remove; delete

Reply:
[11b62ee](https://github.com/surge-synthesizer/clap-saw-demo/pull/19/commits/11b62eeb61a6a1f97bc3d33aaea06e56ee49f039)